Packs/TrendMicroEmailSecurity/Integrations/TrendMicroEmailSecurityEventCollector/README.md

```diff
@@ -0,0 +1,47 @@
+Palo Alto Networks Trend Micro Email Security Event Collector integration for XSIAM.
+
+## Configure Trend Micro Email Security Event Collector on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
```

Comment:
If this is XSIAM and not XSOAR, then the navigation should be:
Navigate to **Settings** > **Configurations** > **Data Collection** > **Automations & Feed Integrations**.